File: Object_Orianted_Programming_For_YTM/Java/ClassExample/src/Araba.java
public class Araba implements Surulebilir{
    public String marka;
    String model; //default olarak public
    private int km; //Encapsulation
    
    public Araba(){}

    public Araba(String marka, String model) {
        this.marka = marka;
        this.model = model;
    }
    public int getKM(){ 
        return km;
    }
    public void calistir() {
        System.out.println(marka + " Araba çalıştı.");
    }
    public void hareketEt() {
        System.out.println("Araba hareket ediyor.");
    }
    public void ucakModuAktif(){
        System.out.println("Araba ucmaya hazirlaniyor");
    }
}

File: Object_Orianted_Programming_For_YTM/Python/ClassExample/src/Araba.py
import Isurulebilir
class Araba(Isurulebilir):
    def __init__(self, marka, model):
        self.marka = marka
        self.model = model
        
    def __init__(self):
        self.__renk = ""
    
    def get_renk(self):
        return self.__renk
    
    def set_renk(self, yeni_renk):
        self.__renk = yeni_renk

    def calistir(self):
        print("Araba çalıştı.")

    def hareket_et(self):
        print("Araba hareket ediyor.")

    def ucakModuAktif(self):
        print("Araba ucmaya hazirlaniyor")